.ci/lint_script.sh

```diff
@@ -6,9 +6,11 @@ source "${CI_DIR}/common.sh"
 
 "${CI_DIR}/helper_shellchecks.sh"
 
+exclude="ffi/sha2.lua"
```

Comment:
Should be done here instead:
https://github.com/koreader/koreader-base/blob/4e9bee31646971cb6db4120f6b2c5009eb46a535/.luacheckrc#L12-L16

Reply:
Yeah, I just learnt it :+1:

Reply:
I should clarify, that's not just for some theoretical organizational reasons, but also for very practical running luacheck locally reasons. ;-)